IABot/checkIfDead.php

```diff
@@ -88,33 +87,14 @@ public function checkDeadlinks( $urls ) {
 		return $returnArray;
 	}
 
+
 	/**
-	 * Function to check whether a given link is dead
-	 * @param string $url URL of the link to be checked
-	 * @return bool True if dead; False if not
-	 * @access public
-	 * @author Niharika Kohli (Niharika29)
```

Comment:
I am pretty certain author attribution goes per-file and not per-function. If there are multiple authors, we just add them to the top of the file. What's the access type? Isn't it given by the function definition itself? Public function?